ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -38,18 +38,29 @@ object LedgerApiTestTool {
   private def exitCode(summaries: Vector[LedgerTestSummary], expectFailure: Boolean): Int =
     if (summaries.exists(_.result.isLeft) == expectFailure) 0 else 1
 
-  private def printAvailableTests(config: Config): Unit = {
+  private def printListOfTests[A](defaults: Seq[A], optionals: Seq[A])(t: A => String): Unit = {
```

Comment:
Can we rename `t` to `getName` or something?

Reply:
Sure.